docs/2_2_common_schema.adoc

```diff
@@ -1205,9 +1205,20 @@ The algebraic relationship to the <<input,`inputs`>> is defined via the <<depend
 An <<outputClock,`output clock`>> can be left unconnected or can be connected to an <<inputClock, `input clock`>>, usually of another FMU (for exceptions see <<triggeredBy>>).
 
 [[local,`local`]]
-`= local`: Local variable that is calculated from other variables or is a continuous-time <<state>> or an event indicator (see <<ModelStructure>>).
+`= local`: Local variables are a subset of all internal variables.
+Local variables are listed in the `modelDescription.xml`.
+Local variables cannot be variables of any other causality (see this table cell).
+They can be continuous-time or discrete <<state,`states`>>, their derivatives, clocked states or event indicators.
+Some local variables are listed in the `<fmiModelDescription><ModelStructure>`, if they are a `Derivative`, `EventIndicator` or `ClockedState`.
+During initialization: you can ignore local variables only if they do not have start values and are not listed as `InitialUnknown`.
+After initialization: local variables are ignorable, if you do not want to use the structure of the model for connectivity or linearization of the FMU (or they are not listed in the `<fmiModelDescription><ModelStructure>` at all).
+You can set a local variable only if they are a state.
```

Comment:
ME only (i would also like it of CS but it was decided no in separate ticket, layered standard should be used

Reply:
Added.